docs/cmake.md

```diff
@@ -297,3 +299,24 @@ OPENDDS_TARGET_SOURCES(messenger
   OPENDDS_IDL_OPTIONS --no-default-nested
 )
 ```
+
+### `OPENDDS_LANGUAGE_MAPPINGS`
+
+After `OPENDDS_TARGET_SOURCES` is used on a target, it will have the
+`OPENDDS_LANGUAGE_MAPPINGS` target property set. This signifies the IDL
+language mappings available in the IDL bindings generated into the target based
+on what is passed to `OPENDDS_IDL_OPTIONS`. It will be a list that can contain
+one or more of the following:
+
+ - `"C++03"`
+   - Default IDL-to-C++ mapping generated by default.
+ - `"C++11"`
+   - IDL-to-C++11 mapping available when passing `-Lc++11`.
+ - `"FACE"`
+   - Will appear if `-GfaceTS` is passed.
```

Comment:
```suggestion
   - Will appear if `-Lface` is passed.
```

Reply:
I based this off what the CMake module is looking for:

https://github.com/objectcomputing/OpenDDS/blob/7cf92aabe0b1d6bd89bddda5db00b8d76ff348d5/cmake/dds_idl_sources.cmake#L161

I can see `opendds_face.mpb` passes both but I'm guessing using `-Lface` would be more correct since it's explicitly dealing with the language mapping?

Reply:
Right, you can (in theory) have one without the other.  Not sure it works in both directions.  Let's use the `-L` options here for consistency.